### PR TITLE
Add config

### PR DESCRIPTION
**Pull request by jonathandick**
This is the initial introduction of a config via esm-module config. In a subsequent PR, I will add the ability to load dynamically non-core widgets.

### CODE REVIEW

File: src/chart-review/chart-review.component.tsx [hunk 1]
```diff
@@ -8,76 +8,86 @@ import Encounters from "./encounters/encounters.component";
...
-        path: `programs`,
-        component: () => {
-          return <Programs />;
-        }
+  const defaultPath = `/patient/${patientUuid}/chart/`;
+
+  function getConfigWidgets() {
+    const w = [];
+    config.widgets.map(widgetName => {
+      w.push(coreWidgets[widgetName]);
```
Comment (fatmali) [inline src/chart-review/chart-review.component.tsx]: map returns a new transformed array, no need of `w` here
```suggestion
return config.widgets.map(widgetName => coreWidgets[widgetName]);
```

File: src/chart-review/chart-review.component.tsx [hunk 2]
```diff
@@ -8,76 +8,86 @@ import Encounters from "./encounters/encounters.component";
...
+    });
+    return w;
+  }
+
+  const coreWidgets = {
```
Comment (fatmali) [inline src/chart-review/chart-review.component.tsx]: can this be stored in a separate file ? Or is it going to come in as a config ?

Comment (jonathandick) [reply src/chart-review/chart-review.component.tsx]: Let me move to a separate file in subsequent PR.

File: src/chart-review/chart-review.component.tsx (hunk 1 above)
Comment (jonathandick) [reply src/chart-review/chart-review.component.tsx]: Done

**Pull request merged**